consensus-types/payload-attribute/getters.go

```diff
@@ -74,3 +74,23 @@ func (a *data) PbV2() (*enginev1.PayloadAttributesV2, error) {
 		Withdrawals:           a.withdrawals,
 	}, nil
 }
+
+// PbV3 returns the payload attribute in version 3.
+func (a *data) PbV3() (*enginev1.PayloadAttributesV3, error) {
+	if a == nil {
+		return nil, errNilPayloadAttribute
+	}
+	if a.version != version.Deneb {
+		return nil, consensus_types.ErrNotSupported("PayloadAttributePbV3", a.version)
+	}
+	if a.timeStamp == 0 && len(a.prevRandao) == 0 {
+		return nil, nil
```

Comment:
Given we are validating fields here, should we check that parent beacon block root and other fields are non-nil here?

Reply:
and non-empty

Reply:
I dont think this check is necessary so ill remove it